### ISSUE TITLE
qBittorrent: Unable to reach torrent client

### ISSUE THREAD

**Issue by Rouzax** [pymedusa/Medusa#8900]
**Describe the bug**
Not sure if this is linked to https://github.com/pymedusa/Medusa/issues/7529 hence the creation of a new ticket.
It looks like the snatch and download went fine.

```
2020-12-18 09:57:52 WARNING  SNATCHQUEUE-SNATCH-361753 :: [d45caa6] qBittorrent: Unable to reach torrent client
```

Medusa Info: | Branch:                 master                  Commit:                 d45caa6b010ad704f4fa84ad120ef4e0b8402e30                  Version:                 0.5.1                  Database:                 44.17
-- | --
Python Version: | 3.8.6 (tags/v3.8.6:db45529, Sep 23 2020, 15:52:53) [MSC v.1927 64 bit (AMD64)]
SSL Version: | OpenSSL 1.1.1g  21 Apr 2020
OS: | Windows-10-10.0.17763-SP0
Locale: | nl_NL.cp1252

Medusa Log
<details>

```
2020-12-18 09:57:51 INFO     SNATCHQUEUE-SNATCH-361753 :: [d45caa6] Beginning to snatch release: The.Mandalorian.S02E08.Chapter.16.The.Rescue.2160p.WEB-DL.DDP5.1.Atmos.x265-MZABI.mkv
2020-12-18 09:57:51 INFO     SNATCHQUEUE-SNATCH-361753 :: [d45caa6] Downloading The.Mandalorian.S02E08.Chapter.16.The.Rescue.2160p.WEB-DL.DDP5.1.Atmos.x265-MZABI.mkv with 8 seeders and 67 leechers and size 6.17 GB from TorrentLeech, through a SearchType.DAILY_SEARCH search
2020-12-18 09:57:52 DEBUG    SNATCHQUEUE-SNATCH-361753 :: [d45caa6] GET URL: https://www.torrentleech.org/download/1752714/The.Mandalorian.S02E08.Chapter.16.The.Rescue.2160p.WEB-DL.DDP5.1.Atmos.x265-MZABI.mkv.torrent [Status: 200]
2020-12-18 09:57:52 DEBUG    SNATCHQUEUE-SNATCH-361753 :: [d45caa6] User-Agent: Medusa/0.5.1 (Windows; 10; d9e905ec-407a-11eb-9caa-00155d02d203)
2020-12-18 09:57:52 DEBUG    SNATCHQUEUE-SNATCH-361753 :: [d45caa6] Calling qBittorrent Client
2020-12-18 09:57:52 DEBUG    SNATCHQUEUE-SNATCH-361753 :: [d45caa6] POST URL: http://download:8080/api/v2/auth/login [Status: 200]
2020-12-18 09:57:52 DEBUG    SNATCHQUEUE-SNATCH-361753 :: [d45caa6] User-Agent: Medusa/0.5.1 (Windows; 10; d9e905ec-407a-11eb-9caa-00155d02d203)
2020-12-18 09:57:52 DEBUG    SNATCHQUEUE-SNATCH-361753 :: [d45caa6] With post data: username=**********&password=**********
2020-12-18 09:57:52 DEBUG    SNATCHQUEUE-SNATCH-361753 :: [d45caa6] GET URL: http://download:8080/api/v2/app/webapiVersion [Status: 200]
2020-12-18 09:57:52 DEBUG    SNATCHQUEUE-SNATCH-361753 :: [d45caa6] User-Agent: Medusa/0.5.1 (Windows; 10; d9e905ec-407a-11eb-9caa-00155d02d203)
2020-12-18 09:57:52 DEBUG    SNATCHQUEUE-SNATCH-361753 :: [d45caa6] qBittorrent: Requested a POST connection to http://download:8080/api/v2/torrents/add with params: None Data: {} Files: {'torrents': b'd8:announce76:https://tracker.torrentleech.org/a/420832d6151d019b6fb4852e28c6f5f2/an...
2020-12-18 09:57:52 DEBUG    SNATCHQUEUE-SNATCH-361753 :: [d45caa6] POST URL: http://download:8080/api/v2/torrents/add [Status: 200]
2020-12-18 09:57:52 DEBUG    SNATCHQUEUE-SNATCH-361753 :: [d45caa6] User-Agent: Medusa/0.5.1 (Windows; 10; d9e905ec-407a-11eb-9caa-00155d02d203)
2020-12-18 09:57:52 DEBUG    SNATCHQUEUE-SNATCH-361753 :: [d45caa6] With post data: --7eb451c06fbf88f0e00b2c27fe049681
 Content-Disposition: form-data; name="torrents"; filename="torr...
2020-12-18 09:57:52 DEBUG    SNATCHQUEUE-SNATCH-361753 :: [d45caa6] qBittorrent: Response to POST request is Ok.
2020-12-18 09:57:52 DEBUG    SNATCHQUEUE-SNATCH-361753 :: [d45caa6] qBittorrent: Requested a POST connection to http://download:8080/api/v2/torrents/resume with params: None Data: {'hashes': '320cf3115b1d088ea11dd88754ff671cc8e1c76e'} Files: None
2020-12-18 09:57:52 DEBUG    SNATCHQUEUE-SNATCH-361753 :: [d45caa6] POST URL: http://download:8080/api/v2/torrents/resume [Status: 200]
2020-12-18 09:57:52 DEBUG    SNATCHQUEUE-SNATCH-361753 :: [d45caa6] User-Agent: Medusa/0.5.1 (Windows; 10; d9e905ec-407a-11eb-9caa-00155d02d203)
2020-12-18 09:57:52 DEBUG    SNATCHQUEUE-SNATCH-361753 :: [d45caa6] With post data: hashes=320cf3115b1d088ea11dd88754ff671cc8e1c76e
2020-12-18 09:57:52 DEBUG    SNATCHQUEUE-SNATCH-361753 :: [d45caa6] qBittorrent: Response to POST request is 
2020-12-18 09:57:52 DEBUG    SNATCHQUEUE-SNATCH-361753 :: [d45caa6] qBittorrent: Requested a POST connection to http://download:8080/api/v2/torrents/setCategory with params: None Data: {'hashes': '320cf3115b1d088ea11dd88754ff671cc8e1c76e', 'category': 'TV'} Files: None
2020-12-18 09:57:52 DEBUG    SNATCHQUEUE-SNATCH-361753 :: [d45caa6] POST URL: http://download:8080/api/v2/torrents/setCategory [Status: 200]
2020-12-18 09:57:52 DEBUG    SNATCHQUEUE-SNATCH-361753 :: [d45caa6] User-Agent: Medusa/0.5.1 (Windows; 10; d9e905ec-407a-11eb-9caa-00155d02d203)
2020-12-18 09:57:52 DEBUG    SNATCHQUEUE-SNATCH-361753 :: [d45caa6] With post data: hashes=320cf3115b1d088ea11dd88754ff671cc8e1c76e&category=TV
2020-12-18 09:57:52 DEBUG    SNATCHQUEUE-SNATCH-361753 :: [d45caa6] qBittorrent: Response to POST request is 
2020-12-18 09:57:52 DEBUG    SNATCHQUEUE-SNATCH-361753 :: [d45caa6] qBittorrent: Requested a POST connection to http://download:8080/api/v2/torrents/increasePrio with params: None Data: {'hashes': '320cf3115b1d088ea11dd88754ff671cc8e1c76e'} Files: None
2020-12-18 09:57:52 DEBUG    SNATCHQUEUE-SNATCH-361753 :: [d45caa6] POST URL: http://download:8080/api/v2/torrents/increasePrio [Status: 409]
2020-12-18 09:57:52 DEBUG    SNATCHQUEUE-SNATCH-361753 :: [d45caa6] User-Agent: Medusa/0.5.1 (Windows; 10; d9e905ec-407a-11eb-9caa-00155d02d203)
2020-12-18 09:57:52 DEBUG    SNATCHQUEUE-SNATCH-361753 :: [d45caa6] With post data: hashes=320cf3115b1d088ea11dd88754ff671cc8e1c76e
2020-12-18 09:57:52 DEBUG    SNATCHQUEUE-SNATCH-361753 :: [d45caa6] The response returned a non-200 response while requesting url http://download:8080/api/v2/torrents/increasePrio Error: HTTPError('409 Client Error: Conflict for url: http://download:8080/api/v2/torrents/increasePrio')
2020-12-18 09:57:52 WARNING  SNATCHQUEUE-SNATCH-361753 :: [d45caa6] qBittorrent: Unable to reach torrent client
2020-12-18 09:57:52 INFO     SNATCHQUEUE-SNATCH-361753 :: [d45caa6] qBittorrent: Unable to set torrent priority because torrent queueing is disabled in qBittorrent settings.
2020-12-18 09:57:52 DEBUG    SNATCHQUEUE-SNATCH-361753 :: [d45caa6] Checking if we should stop searching for a better quality for for episode The.Mandalorian.S02E08.Chapter.16.The.Rescue.2160p.WEB-DL.DDP5.1.Atmos.x265-MZABI.mkv
2020-12-18 09:57:52 DEBUG    SNATCHQUEUE-SNATCH-361753 :: [d45caa6] Found codec for The Mandalorian Chapter 16
2020-12-18 09:57:52 DEBUG    SNATCHQUEUE-SNATCH-361753 :: [d45caa6] 361753: Formatting pattern: %SN - %Sx08 - %EN - %QN -> The Mandalorian - 2x08 - Chapter 16 - 4K UHD WEB-DL
2020-12-18 09:57:52 DEBUG    SNATCHQUEUE-SNATCH-361753 :: [d45caa6] Transaction with 1 queries executed
```
</details>

qBittorrent Log
<details>

```
(N) 2020-12-18T09:51:32 - WebAPI login success. IP: ::ffff:192.168.2.16
(N) 2020-12-18T09:56:32 - WebAPI login success. IP: ::ffff:192.168.2.16
(N) 2020-12-18T09:57:52 - WebAPI login success. IP: fe80::6101:d490:a182:65ec%4
(N) 2020-12-18T09:57:52 - 'The.Mandalorian.S02E08.Chapter.16.The.Rescue.2160p.WEB-DL.DDP5.1.Atmos.x265-MZABI.mkv' added to download list.
(N) 2020-12-18T10:00:56 - Torrent: The.Mandalorian.S02E08.Chapter.16.The.Rescue.2160p.WEB-DL.DDP5.1.Atmos.x265-MZABI.mkv, running external program, command: powershell "C:\GitHub\TorrentScript\TorrentScript.ps1" -DownloadPath 'C:\TEMP\Torrent\Downloads' -DownloadName 'The.Mandalorian.S02E08.Chapter.16.The.Rescue.2160p.WEB-DL.DDP5.1.Atmos.x265-MZABI.mkv' -DownloadLabel 'TV' -TorrentHash '320cf3115b1d088ea11dd88754ff671cc8e1c76e'
(N) 2020-12-18T10:01:32 - WebAPI login success. IP: ::ffff:192.168.2.16
```
</details>

**Comment by medariox** [comment]
Whatever you are running in qBittorrent is conflicting with increase priority command. Disable the script or the high priority from medusa.

**Comment by Rouzax** [comment]
Not entirely sure what you mean with disable script or disable high priority in Medusa.
I've changed nothing on my setup and this is the first I see this.

**Comment by medariox** [comment]
Yes, that's what I mean.
You are not running a script within qBittorrent?
```
(N) 2020-12-18T10:00:56 - Torrent: The.Mandalorian.S02E08.Chapter.16.The.Rescue.2160p.WEB-DL.DDP5.1.Atmos.x265-MZABI.mkv, running external program, command: powershell "C:\GitHub\TorrentScript\TorrentScript.ps1" -DownloadPath 'C:\TEMP\Torrent\Downloads' -DownloadName 'The.Mandalorian.S02E08.Chapter.16.The.Rescue.2160p.WEB-DL.DDP5.1.Atmos.x265-MZABI.mkv' -DownloadLabel 'TV' -TorrentHash '320cf3115b1d088ea11dd88754ff671cc8e1c76e'
```
`C:\GitHub\TorrentScript\TorrentScript.ps1`

We also haven't changed that code since ages.

**Comment by Rouzax** [comment]
I run a script when the download is finished, but have always done that

**Comment by medariox** [comment]
So the script was probably triggered at the same time or something. Likely happens only once or twice every year.

**Comment by Rouzax** [comment]
Just to understand, the `allow high priority` is to add downloads to qBittorrent with a higher priority?
If that is all I can just disable that as I also don't need it.

**Comment by medariox** [comment]
Yes, that's all it does.
